test/cpp/end2end/grpc_authz_end2end_test.cc

```diff
@@ -823,6 +1150,11 @@ TEST_F(GrpcAuthzEnd2EndTest, FileWatcherRecoversFromFailure) {
   status = SendRpc(channel, &context2, &resp2);
   EXPECT_TRUE(status.ok());
   EXPECT_EQ(resp2.message(), kMessage);
+  EXPECT_EQ(audit_logs_.size(), 2);
```

Comment:
Suggest using `ElementsAre()` here as well.  Note that you can say `audit_logs_.clear()` after the previous pass to clear out the entry you've already checked.

Reply:
Done.